tools/guix-env.scm

```diff
@@ -0,0 +1,22 @@
+;; Environment manifest to build with guix, invoke
```

Comment:
What is this file for?

Reply:
I'll remove it, I used it when developping